sentences/ca/_common.yaml

```diff
@@ -1,12 +1,57 @@
 language: ca
 responses:
   errors:
-    no_intent: TODO Sorry, I couldn't understand that
-    no_area: TODO No area named {{ area }}
-    no_domain: TODO {{ area }} does not contain a {{ domain }}
-    no_device_class: TODO {{ area }} does not contain a {{ device_class }}
-    no_entity: TODO No device or entity named {{ entity }}
-    handle_error: TODO An unexpected error occurred while handling the intent
-lists: {}
-expansion_rules: {}
-skip_words: []
+    no_intent: Ho sento, no entenc això
+    no_area: No hi ha l'àrea {{ area }}
+    no_domain: No hi ha {{ domain }} a {{ area }}
+    no_device_class: No hi ha {{ device_class }} a {{ area }}
+    no_entity: No hi ha cap dispositiu o entitat que es digui {{ entity }}
+    handle_error: He trobat un error inesperat mentre feia l'intent
+lists:
+  color:
+    values:
+      - blanc
+      - negre
+      - vermell
+      - taronja
+      - groc
+      - verd
+      - blau
+      - porpra
+      - marró
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "celsius"
+      - "fahrenheit"
+      - in: "f"
+        out: "fahrenheit"
+
+expansion_rules:
+  name: "[el | la | l'] {name}"
+  area: "[en el | en la | en | del | de la | al | a la | a] {area}"
+  what_is: "(quin és | quina és | que es)"
+  brightness: "{brightness:brightness_pct} [per cent]"
+  turn_on: "(encén | engega | posa)"
```

Comment:
turn_on should be translated etc

Reply:
So `expansion_rules` "action" should have the language name action, rather than the English one?
I'm guessing in the future there will be other common names (besides `name` and `area`), but i'm ok to follow that 👍🏻

Reply:
On my phone but we have this documented in sentences/README.